depends/packages/libgmp.mk

```diff
@@ -1,11 +1,10 @@
 package=libgmp
-$(package)_download_path=https://github.com/z-classic/$(package)/archive/
-$(package)_file_name=$(package)-$($(package)_git_commit).tar.gz
-$(package)_download_file=$($(package)_git_commit).tar.gz
-$(package)_sha256_hash=59b2c2b5d58fdf5943bfde1fa709e9eb53e7e072c9699d28dc1c2cbb3c8cc32c
-$(package)_git_commit=aece03c7b6967f91f3efdac8c673f55adff53ab1
+$(package)_version=6-1-2
+$(package)_download_path=https://github.com/ezzygarmyz/libgmp/releases/download/6.1.2/
```

Comment:
I'm not sure its a good idea to depend on your git repo... Maybe we should fork it from on of the community git.

Reply:
Yes, it would definitely be better. This was my experience when I wanted to build a full node.